### ISSUE TITLE
using pre-trained model on mel-spectrogram not seen during training?

### ISSUE THREAD

**Issue by yxt132** [NVIDIA/waveglow#84]
Has anyone tried to use the pre-trained model to synthesize audio from the mel-spectrogram generated by a model which was trained separately on a different corpus? For example, the pre-trained waveglow model was trained using ljspeech corpus with sampling rate of 22050 hz. And the model (Tacotron-2) which was used to create the mel-spectrogram was trained with a different corpus (say male's voice) with the same sampling rate. Can the waveglow successfully generate the male's voice? I am wondering if waveglow will generalize well or I have to train waveglow everytime when I want to generate a new voice? Because it takes really long time to train waveglow, it will be inconvenient because waveglow has to be trained for different training corpus. In addition, has anyone tried to train waveglow on multi-speaker corpus and was it successful?

**Comment by rafaelvalle** [comment]
We did some quick experiments and a waveglow trained on LJS works ok with a female voice that is not LJS.

**Comment by yxt132** [comment]
Thanks, Rafael! By "finetune", do you mean transfer learning or something else?

**Comment by rafaelvalle** [comment]
By finetune I mean training on the new dataset but starting with weights from a pre-trained model.

**Comment by tugstugi** [comment]
LJSpeech trained waveglow can even generate audio for a foreign male voice. The attached male voice is Mongolian and was trained only for 10K steps with Tacotron2 and synthesized with the pretrained LJSpeech waveglow.

[10k_mongolian_ljspeech_vocoder.zip](https://github.com/NVIDIA/waveglow/files/2907078/10k_mongolian_ljspeech_vocoder.zip)

**Comment by rafaelvalle** [comment]
@tugstugi did you use the pretrained Taco2 for training 10K steps on this dataset?

**Comment by tugstugi** [comment]
@rafaelvalle is it possible to provide an updated pretrained LJSpeech model mirroring the current code here? With the current available pretrained model, the finetuning seems to be not possible.

**Comment by rafaelvalle** [comment]
Here's the link to the model with 256 channels trained on LJS.
https://drive.google.com/open?id=1WsibBTsuRg_SF2Z6L6NFRTT-NjEy1oTx

**Comment by rafaelvalle** [comment]
Closing due to inactivity.

**Comment by ricktjwong** [comment]
> LJSpeech trained waveglow can even generate audio for a foreign male voice. The attached male voice is Mongolian and was trained only for 10K steps with Tacotron2 and synthesized with the pretrained LJSpeech waveglow.

@tugstugi Can I confirm that this was a Mongolian voice trained from scratch with Taco2 and synthesised with pertained LJSpeech waveglow? (instead of fine-tuning from the LJSpeech waveglow with the Mongolian dataset).

**Comment by laurentbenaroya** [comment]
I'm also trying to fine-tune a new English voice. How much audio do I need for this to work? Are there any modifications to be made for training, such as a learning rate schedule? I'm starting with a pre-trained model.
